src/garage/envs/dm_control/dm_control_env.py

```diff
@@ -22,59 +21,161 @@ def _flat_shape(observation):
     return np.sum(int(np.prod(v.shape)) for k, v in observation.items())
 
 
-class DmControlEnv(gym.Env):
+class DMControlEnv(Environment):
     """Binding for `dm_control <https://arxiv.org/pdf/1801.00690.pdf>`."""
 
-    def __init__(self, env, name=None):
-        """Create a DmControlEnv.
+    def __init__(self, env, name=None, max_episode_length=None):
```

Comment:
is there any instance of a DMControlEnv without an episode length? I don't think there is, in which case this can be removed.